packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -425,7 +439,29 @@ export const conditionallyJoinExpressions = (
  * @param meta Plugin metadata
  */
 export const buildStyledComponent = (tag: Tag, cssOutput: CSSOutput, meta: Metadata): t.Node => {
-  const { sheets, classNames } = transformCss(cssOutput.css.map((x) => getItemCss(x)).join(''));
+  const unconditionalCss: string[] = [];
+  const logicalCss: Array<CssItem> = [];
```

Comment:
Is this shorthand `CssItem[]` not preferred here?

Reply:
- [ ] use shorthand